changelog.txt

```diff
@@ -36,6 +36,14 @@ Template for new versions:
 
 ## Removed
 
+# 50.13-r2
+
+## New Features
+- `gui/autobutcher`: add functionality to butcher and unbutcher all animals
+
+## Misc Improvements
+- `gui/autobutcher`: add warning if autobutcher is disabled
+
```

Comment:
we don't add a version header to the changelog until we tag the release. it's possible that the version this goes into will not be named `50.13-r2`, even that's what the current plan is. These should go under the `Future` section.